.github/integration/scripts/make_certs.sh

```diff
@@ -1,14 +1,17 @@
 #!/bin/sh
+set -e
 
-# install openssl if it's missing
-if [ ! "$(command -v openssl)" ]; then
-    if [ "$(id -u)" != 0 ]; then
-        echo "openssl is missing, unable to install it"
-        exit 1
-    fi
+# install requirements if it's missing
+for r in openssl openjdk8-jre-base; do
+    if [ ! "$(command -v openssl)" ]; then
```

Comment:
```suggestion
    if [ ! "$(command -v "$r")" ]; then
```
? Even though openjdk8-jre-base seems an unexpected name to me.